src/explore-education-statistics-admin/src/pages/release/footnotes/ReleaseFootnotesPage.tsx

```diff
@@ -69,17 +85,33 @@ const ReleaseFootnotesPage = ({
     return (
       <>
         {canUpdateRelease && (
-          <ButtonLink
-            to={generatePath<ReleaseRouteParams>(
-              releaseFootnotesCreateRoute.path,
-              {
-                publicationId,
-                releaseId,
-              },
+          <div className="dfe-flex dfe-justify-content--space-between">
+            <ButtonLink
+              to={generatePath<ReleaseRouteParams>(
+                releaseFootnotesCreateRoute.path,
+                {
+                  publicationId,
+                  releaseId,
+                },
+              )}
+            >
+              Create footnote
+            </ButtonLink>
+
+            {footnotes.length > 1 && (
+              <Button
+                variant="secondary"
```

Comment:
Would be tempted to make this primary whilst the reordering is happening. Combined with the above change to hide the 'Create footnote' button, this should make it really obvious what the user has to do.